docs/search/indexing/sparse.mdx

```diff
@@ -11,6 +11,19 @@ SPLADE outputs sparse vectors with over 30,000 entries.
 
 A sparse HNSW index is a specialized HNSW index designed to handle sparse vectors.
 
+## Using `pg_sparse`
+
+Paradedb introduces `pg_sparse`, a fork of `pgvector`, for sparse vectors. To get started,
+ensure the extension is installed by following the [installation guide](https://github.com/paradedb/paradedb/blob/dev/pg_sparse/README.md#installation).
+
+The following adds a sparse vector column using `svector`:
+
+```sql
+ALTER TABLE mock_items ADD COLUMN sparse_embedding svector(30522);
+```
+
+For additional details, please refer to the [Github README](https://github.com/paradedb/paradedb/blob/dev/pg_sparse/README.md).
+
```

Comment:
```suggestion
ParadeDB uses the `pg_sparse` sparse HNSW index, a fork of `pgvector` designed for sparse vector search designed and maintained by ParadeDB.
```

Reply:
Cutting down all the text is fine; I still feel like mentioning `svector` somewhere on this page would be wise. 
It was a point of confusion for me, when I first started my testing. 
But you make the final call on this.

Reply:
I'm confused -- we have documentation on showing how to create the `shnsw` index. That is the recommended approach for interacting with the index

Reply:
`pg_sparse` creates the `svector` type. I understood, from reading the docs, that a new system was powering this sparse HNSW. However, I did not realize from the docs alone that the `svector` even existed. And was required.

Maybe an extra line like
> ParadeDB uses the `pg_sparse` sparse HNSW index, a fork of `pgvector` designed for sparse vector search designed and maintained by ParadeDB. 
`pg_sparse` introduces the `svector` type, replacing `vector`, for sparse vectors.

`svector` is not mentioned anywhere in the docs, except for in the casting section. But without understanding what it is, there is no way to understand why you would even need to cast. Or that you need to use it to construct a sparse embed